main.py

```diff
@@ -1,7 +1,30 @@
+# from venv import create
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    for i in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=i)
+    actors = ["George Klooney",
+              "Kianu Reaves",
+              "Scarlett Keegan",
+              "Will Smith",
+              "Jaden Smith",
+              "Scarlett Johansson"]
+    for i in actors:
+        a_name, surname = i.split(" ")
```

Comment:
Ensure that all actor names in the list have exactly two parts (first name and last name) to avoid errors during the split operation.